sdk/digitaltwins/azure-digitaltwins-core/src/samples/java/com/azure/digitaltwins/core/DigitalTwinsLifecycleSample.java

```diff
@@ -0,0 +1,118 @@
+
+public class DigitalTwinsLifecycleSample {
+    private static final ClientLogger logger = new ClientLogger(DigitalTwinsLifecycleSample.class);
+
+    private static final String tenantId = System.getenv("TENANT_ID");
+    private static final String clientId = System.getenv("CLIENT_ID");
+    private static final String clientSecret = System.getenv("CLIENT_SECRET");
+    private static final String endpoint = System.getenv("DIGITAL_TWINS_ENDPOINT");
+
+    private static final int MaxTimeForTwinOperationsInSeconds = 10;
+
+    private static final URL DtdlDirectoryUrl = DigitalTwinsLifecycleSample.class.getClassLoader().getResource("DTDL");
+    private static final Path DtDlDirectoryPath;
+    private static final Path TwinsPath;
+    private static final Path ModelsPath;
+    private static final Path RelationshipsPath;
+
+    private static final DigitalTwinsAsyncClient client;
+
+    static {
+        try {
+            assert DtdlDirectoryUrl != null;
+            DtDlDirectoryPath = Paths.get(DtdlDirectoryUrl.toURI());
+        } catch (URISyntaxException e) {
+            logger.error("Unable to convert the DTDL directory URL to URI: ", e);
+            throw new RuntimeException(e);
+        }
+        TwinsPath = Paths.get(DtDlDirectoryPath.toString(), "DigitalTwins");
+        ModelsPath = Paths.get(DtDlDirectoryPath.toString(), "Models");
+        RelationshipsPath = Paths.get(DtDlDirectoryPath.toString(), "Relationships");
+
+        client = new DigitalTwinsClientBuilder()
+            .tokenCredential(
+                new ClientSecretCredentialBuilder()
+                    .tenantId(tenantId)
+                    .clientId(clientId)
+                    .clientSecret(clientSecret)
+                    .build()
+            )
+            .endpoint(endpoint)
+            .httpLogOptions(
+                new HttpLogOptions()
+                    .setLogLevel(HttpLogDetailLevel.BODY_AND_HEADERS))
+            .buildAsyncClient();
+    }
+
+    public static void main(String[] args) throws IOException, InterruptedException {
+        // Ensure existing twins with the same name are deleted first
+        deleteTwins();
+
+        // Create twin counterparts for all the models
+        createTwins();
+    }
+
+    public static void deleteTwins() throws IOException, InterruptedException {
+        System.out.println("DELETE DIGITAL TWINS");
+        Map<String, String> twins = FileHelper.loadAllFilesInPath(TwinsPath);
+        final Semaphore deleteTwinsSemaphore = new Semaphore(0);
+
+        twins
+            .forEach((twinId, twinContent) -> client.deleteDigitalTwin(twinId)
+                .subscribe(
+                    aVoid -> { },
+                    throwable -> {
+                        // If digital twin does not exist, ignore.
+                        if (!(throwable instanceof ErrorResponseException) || !((ErrorResponseException) throwable).getValue().getError().getCode().equals("DigitalTwinNotFound")) {
```

Comment:
How to access response status code? `ErrorResponse.getValue().getError().getCode()` is string.